src/api/tests/test_gfcr.py

```
@@ -359,24 +360,287 @@ def test_notes_in_report_export(
             assert row[6] == "F7 test notes", f"F7 notes not found in row: {row}"
             break
 
-    # Check BusinessesFinanceSolutions sheet - should have notes in column 12
-    bfs_sheet = wb["BusinessesFinanceSolutions"]
+    # Check FacilitiesSolutions sheet - notes are now at column 16 (index 15)
+    bfs_sheet = wb["FacilitiesSolutions"]
     bfs_row = list(bfs_sheet.iter_rows(min_row=2, max_row=2, values_only=True))[0]
```

Ruff (0.15.13)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)